error: 'invalid_argument' is not a member of 'std' (GCC 4.7.2)

```
In file included from crypto/equihash.cpp:15:0:
./crypto/equihash.h: In function 'bool EhBasicSolve(unsigned int, unsigned int, const eh_HashState&, std::function<bool(std::vector<unsigned char>)>, std::function<bool(EhSolverCancelCheck)>)':
./crypto/equihash.h:227:15: error: 'invalid_argument' is not a member of 'std'
./crypto/equihash.h: In function 'bool EhOptimisedSolve(unsigned int, unsigned int, const eh_HashState&, std::function<bool(std::vector<unsigned char>)>, std::function<bool(EhSolverCancelCheck)>)':
./crypto/equihash.h:251:15: error: 'invalid_argument' is not a member of 'std'
```

Needs to include `<stdexcept>`.

--- a/src/crypto/equihash.h
+++ b/src/crypto/equihash.h
@@ -13,6 +13,7 @@
 
 #include <cstring>
 #include <exception>
+#include <stdexcept>
 #include <functional>
 #include <memory>
 #include <set>

